extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-postgresql/src/main/java/org/apache/guacamole/auth/postgresql/conf/PostgreSQLEnvironment.java

```diff
@@ -115,6 +115,12 @@ public class PostgreSQLEnvironment extends JDBCEnvironment {
      * The default value to use for SSL mode if none is explicitly configured.
      */
     private final PostgreSQLSSLMode DEFAULT_SSL_MODE = PostgreSQLSSLMode.PREFER;
+    
+    /*
+     * The default maximum number of identifiers/parameters to be included in a 
+     * single batch when executing SQL statements.
+     */
+    private static final int DEFAULT_BATCH_SIZE = 2147483647;
```

Comment:
Is there documentation that might be referenced here that shows `2147483647` is the relevant limit for PostgreSQL?